make/test/BuildTestSetupAOT.gmk

```diff
@@ -55,7 +55,11 @@ TARGETS += $(BUILD_SETUP_AOT)
 $(eval $(call SetupCopyFiles, COPY_SETUP_AOT, \
     SRC := $(SETUP_AOT_SUPPORT)/classes, \
     DEST := $(TEST_IMAGE_DIR)/setup_aot, \
-    FILES := TestSetupAOT.class, \
+    FILES := JavacBenchApp.class \
+             JavacBenchApp$$ClassFile.class \
+             JavacBenchApp$$FileManager.class \
+             JavacBenchApp$$SourceFile.class \
+             TestSetupAOT.class, \
```

Comment:
Listing all class files like this is cumbersome. Perhaps we should build this into a jar file instead now that there are more classes? It should be as easy as adding `JAR := <name of jar>` to the SetupJavaCompilation call above and then copy that file.

Reply:
I changed the output to a JAR file so there's no need to enumerate all classes. When running the tool, I need to extract the class files to `$$($1_AOT_JDK_OUTPUT_DIR)` due to the reasons described in RunTests.gmk.